.travis.yml

```diff
@@ -37,6 +37,12 @@ jobs:
       sudo: required
       env: TOXENV=py37-ansi2html
 
+    -
+      python: 3.8
+      dist: xenial
+      sudo: required
```

Comment:
`sudo` is apparently default now (at least for `xenial`), so you can remove this line.

Reply:
done